### PR TITLE
Tutorial Essential genes detection with Transposon insertion sequencing 

### PR DESCRIPTION
**Pull request by Delphine-L**
Tutorial on prediction of essential genes, including pre-processing and use of Transit tool. 
The Tutorial is mostly complete, so I would be grateful to have returns on it. 
I still have to make some tools installations and to generate a workflow.

### CODE REVIEW

**Comment by Delphine-L** [comment]
@bebatut This tutorial is ready for review

**Comment by bebatut** [comment]
Thanks a lot @Delphine-L. I will review it asap 😄

**Comment by erasche** [comment]
@Delphine-L could you rebase this against master? We can update it to use the new bibliography feature from #1324 :)  

I wrote a small patch before I realised it wouldn't build: [0001-bibliography.patch](https://github.com/galaxyproject/training-material/files/3116583/0001-bibliography.patch.txt)

File: topics/genome-annotation/tutorials/tnseq/slides.html [hunk 1]
```diff
@@ -0,0 +1,519 @@
...
+  - "How to get TA sites Coverage ? "
+  - "How to predict essential genes ?"
+objectives:
+  - "Understand the read structure of TnSeq analyses"
+  - "Predict Essential genes with Transit"
+  - "Compare gene essentiality in control sample and an different experimental condition"
+time_estimation: "7h"
+key_points:
+contributors:
+  - dlariviere
```
Comment (bebatut) [inline topics/genome-annotation/tutorials/tnseq/slides.html]: It should be `delphine-l` here (as in the `CONTRIBUTORS.yaml` file)

File: topics/genome-annotation/tutorials/tnseq/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,681 @@
...
+  - essential genes
+questions:
+  - "What is Transposon insertion Sequencing?"
+  - "How to get TA sites Coverage ? "
+  - "How to predict essential genes ?"
+objectives:
+  - "Understand the read structure of TnSeq analyses"
+  - "Predict Essential genes with Transit"
+  - "Compare gene essentiality in control sample and an different experimental condition"
+time_estimation: "7h"
```
Comment (bebatut) [inline topics/genome-annotation/tutorials/tnseq/tutorial.md]:
```suggestion
time_estimation: "7H"
```

File: topics/genome-annotation/tutorials/tnseq/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,681 @@
...
+key_points:
+contributors:
+  - dlariviere
```
Comment (bebatut) [inline topics/genome-annotation/tutorials/tnseq/tutorial.md]: It should be `delphine-l` here (as in the `CONTRIBUTORS.yaml` file)

File: topics/genome-annotation/tutorials/tnseq/slides.html [hunk 4]
```diff
@@ -0,0 +1,519 @@
...
+  - essential genes
+questions:
+  - "What is Transposon insertion Sequencing?"
+  - "How to get TA sites Coverage ? "
+  - "How to predict essential genes ?"
+objectives:
+  - "Understand the read structure of TnSeq analyses"
+  - "Predict Essential genes with Transit"
+  - "Compare gene essentiality in control sample and an different experimental condition"
+time_estimation: "7h"
```
Comment (bebatut) [inline topics/genome-annotation/tutorials/tnseq/slides.html]:
```suggestion
time_estimation: "7H"
```

File: topics/genome-annotation/tutorials/tnseq/tutorial.md (hunk 3 above)
Comment (Delphine-L) [reply topics/genome-annotation/tutorials/tnseq/tutorial.md]: I managed to make mistakes on my own name tag 🙄 Thanks !

File: topics/genome-annotation/tutorials/tnseq/slides.html (hunk 4 above)
Comment (Delphine-L) [reply topics/genome-annotation/tutorials/tnseq/slides.html]: It works now, Thank You !

**Comment by Delphine-L** [comment]
Hi @bebatut ! I completed the to-do list, fixed some typos... Let me know if it works for you, Thanks!

**Comment by bebatut** [comment]
Thanks a lot @Delphine-L for this nice tutorial!

**Pull request merged**